core/domain/exp_services.py

```diff
@@ -2079,6 +2080,9 @@ def compute_models_to_put_when_saving_new_exp_version(
         )
     )
 
+    voiceover_services.update_exploration_voice_artist_link_model(
```

Comment:
The method updates the [exploration voice artists link model](https://github.com/oppia/oppia/blob/6b3f94d4400b37e27f884591295045cdcfe51d95/core/storage/voiceover/gae_models.py#L324) data for an exploration after the [MR jobs](https://github.com/oppia/oppia/blob/6b3f94d4400b37e27f884591295045cdcfe51d95/core/jobs/batch_jobs/manual_voice_artist_name_job.py#L51) have been triggered. This ensures the model always be consistent with the latest exploration data even after successful run of MR job.